### PR TITLE
Update mirror-google-docs.yaml

### PR DESCRIPTION
**Pull request by Bramwelian**

### CODE REVIEW

File: .github/workflows/mirror-google-docs.yaml [hunk 1]
```diff
@@ -8,22 +8,22 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout gh-pages
+    - name: BramwelIanAuku
       uses: actions/checkout@v3
-      with:
```
[the remark this answers is not in the document]
Comment (Bramwelian) [reply .github/workflows/mirror-google-docs.yaml:13]: Determine to tackle malware and debugs for systems hiccups

File: .github/workflows/mirror-google-docs.yaml [hunk 2]
```diff
@@ -8,22 +8,22 @@ jobs:
...
+      with:Githubandapi
         ref: gh-pages
         path: gh-pages
-    - name: Publish to github pages
-      run: |
```
Comment (Bramwelian) [inline .github/workflows/mirror-google-docs.yaml:17]: Cloud code microsoft

**Review by Bramwelian — COMMENTED** [review]
Building a good workforce community group

**Comment by rscohn2** [comment]
Looks like you targeted the wrong repo.

**Pull request closed without merging**